WTWSMS/history/titles/e_persia.txt

```diff
@@ -8,11 +8,29 @@
 	law = centralization_3
 }
 11.1.1={
-	holder=159546
+	holder=159546 
```

Comment:
Which monarch is he? Tried to find him on Wikipedia but failed.

Reply:
Yes, i already fixed it.